### PR TITLE
FEATURE-1: first commit

### PR DESCRIPTION
**Pull request by roninkuma**

### CODE REVIEW

File: src/main/java/com/java_essential/SinRoman/lesson_2/Main.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
+package com.java_essential.SinRoman.lesson_2;
+
+public class Main {
+    public static void main(String[] args) {
+
```
Comment (ochedenis) [inline src/main/java/com/java_essential/SinRoman/lesson_2/Main.java:5]: Clear this empty space.

File: src/main/java/com/java_essential/SinRoman/lesson_2/Main.java [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        if (student.getEmail() == null) {
+            varEmail++;
+        }
+        if (student.getUsername() == null) {
+            varUserName++;
+        }
+        System.out.println("The age " + varAge);
+        System.out.println("Student no show emil " + varEmail);
+        System.out.println("Student no show username " + varUserName);
+
```
Comment (ochedenis) [inline src/main/java/com/java_essential/SinRoman/lesson_2/Main.java:60]: Clear this empty space.

File: src/main/java/com/java_essential/SinRoman/lesson_2/Main.java [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    }
+
```
Comment (ochedenis) [inline src/main/java/com/java_essential/SinRoman/lesson_2/Main.java:62]: Clear this empty space.

File: src/main/java/com/java_essential/SinRoman/lesson_2/Student.java [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+public class Student {
+
+    private long id;
+    private String username;
+    private String email;
+    private String phone;
+    private String name;
+    private String lastName;
+    private int age;
+
```
Comment (ochedenis) [inline src/main/java/com/java_essential/SinRoman/lesson_2/Student.java:12]: Clear this empty space.

File: src/main/java/com/java_essential/SinRoman/lesson_2/Student.java [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+
```
Comment (ochedenis) [inline src/main/java/com/java_essential/SinRoman/lesson_2/Student.java:13]: Clear this empty space.

**Review by ochedenis — CHANGES_REQUESTED** [review]

**Review by yuriy-tsarenko — COMMENTED** [review]
I'm looking forward for your fixing commits Roman)

**Comment by yuriy-tsarenko** [comment]
Task not completed

**Pull request closed without merging**